### ISSUE TITLE
Add action to postfetch function params

### ISSUE THREAD

**Issue by jakeaaron** [lexich/redux-api#166]
We already get request options in the function, but I am wanting to retry an action if the fetch fails. If we had the action in the postfetch I could just dispatch the action again.

Is there a different way to retry an action if it fails?

**Comment by lexich** [comment]
Hi @jakeaaron Very interesting question. I think that at this moment postfetch is only way.

**Comment by lexich** [comment]
But it's good feature request

**Comment by jakeaaron** [comment]
Thanks for the response @lexich. 
I don't think it's possible with postfetch at this moment, because we're not getting the action type in the function.
Is this correct?

**Comment by lexich** [comment]
There is another way. You can use https://github.com/lexich/redux-api/blob/master/docs/DOCS.md#responsehandler and implement here retry logic

**Comment by jakeaaron** [comment]
Oh! I didn't realize... you can dispatch an action in responseHandler?

**Comment by lexich** [comment]
Yes responsehandler is bad idea in this case. But you can write custom adapter and you'll have access to request properties.

**Comment by jakeaaron** [comment]
2 questions:

1. Does post fetch not get called on a fetch failure?
2. Do you think it would be sufficient to just return the action type since we're already returning the request?

**Comment by lexich** [comment]
1. Yes, postfetch called only on success.
2. I didn't understand, what's you mean, maybe example?
